### ISSUE TITLE
is it possible to change client name color?

### ISSUE THREAD

**Issue by CryRocks** [AikoMidori/TeamSpeak-5-Dark#4]
Hi,

i wanna know if i can change the textcolor from client names somehow? 
since i changed the background and made it to show in channel tree and in some places its white text on white/bright background.

I did some testing but couldn't figure out a way to do it. Would appreciate the help! :)

**Comment by qeinz** [comment]
could you provide a screenshot?

**Comment by CryRocks** [comment]
sure, on the third user, you can see it the most

![grafik](https://github.com/AikoMidori/TeamSpeak-5-Dark/assets/107412582/a846e75a-9e1a-4532-b1cc-fec73654147d)

**Comment by qeinz** [comment]
ahh i see, i think the best way is, if you go into debug mode and look into ur browser, than look for the element name, so you can change the color on the code

**Comment by CryRocks** [comment]
yeah thats what i already tried, but didn't really found a way to do it

**Comment by qeinz** [comment]
look into the css file 


  --name-color: var(--white) !important;
   

![image](https://github.com/AikoMidori/TeamSpeak-5-Dark/assets/66924611/02330a51-16a1-47d2-b7a5-13018cdeeda1)

**Comment by qeinz** [comment]
change the --white color i think

**Comment by CryRocks** [comment]
yeah, i already changed that to red for example but it doesn't change the name color
not sure why

**Comment by qeinz** [comment]
```css
/* name color */
.ts-server-tree-item-leaf.client, .ts-server-tree-item-leaf.client.self {
    color: var(--name-color) !important;
}
```


look for this

**Comment by qeinz** [comment]
im using the darkNue.css

**Comment by CryRocks** [comment]
and it changes the client name color for you if you edit them? 
since i changed it directly to a color and it doesn't change anything, tried the var nothing happens
the client names are still white

i guess this:
![grafik](https://github.com/AikoMidori/TeamSpeak-5-Dark/assets/107412582/5b5e515e-17e8-49bc-89d3-677221cb5902)
is overwriting them or something else idk

**Comment by qeinz** [comment]
yea i think you have to change this, lets have a look in debug console in browser, there you can see more info, got no time rn

**Comment by CryRocks** [comment]
so i have to deactivate the first color stuff, wich i showed in my last message
and then i also have to deactivate --text-color to even get the --name-color to work on only names

weird

**Comment by qeinz** [comment]
yes, unfortunately the code behind is written a bit "funny"....

**Comment by CryRocks** [comment]
yeah i see it

got it to work now, thanks! now i just have to find a good color wich is readable and stays that way on darker/brighter background.. and match the purple theme i guess

(i also had to edit a couple of other elements, since some had a weird color after i removed both text and the option above, like text sender in chat had a green name, every normal white color was gray etc.)

![grafik](https://github.com/AikoMidori/TeamSpeak-5-Dark/assets/107412582/766f6d70-9877-401d-9279-45f462299032)
even got it to work with a nice little glow effekt, now its readable :D

**Comment by qeinz** [comment]
great work!

hope we will see an update on this theme soon.... there are so many bugs....
